product_docs/docs/pgd/5/reference/nodes-management-interfaces.mdx

```diff
@@ -235,11 +235,11 @@ bdr.alter_node_option(node_name text,
 -   `config_value` &mdash; New value to be set for the given key.
 
 The node options that can be changed using this function are:
-- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is '-1'.
-- `route_fence` &mdash; Whether the node is fenced from routing; when true, the node can't receive connections from PGD Proxy. Default is 'f' (false).
-- `route_writes` &mdash; Whether writes can be routed to this node, that is, whether the node can become write leader. Default is 't' (true) for data nodes and 'f' (false) for other node types.
-- `route_reads` &mdash; Whether read-only connections can be routed to this node. Currently reserved for future use. Default is 't' (true) for data and subscriber-only nodes, 'f' (false) for witness and standby nodes.
-- `route_dsn` &mdash; The dsn that the proxy will use to connect to this node. This is optional; if not set it defaults to the node's node_dsn value.
+- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is `-1`.
```

Comment:
```suggestion
- `route_priority` &mdash; Relative routing priority of the node against other nodes in the same node group. Default is `'-1'`.
```
Okay, I've sometimes seen the single quotes used when it was meant to be back ticks, so yes, using both makes it clearer (to me at least!) that the single quotes are part of the value. I'll fix the rest of these.